.gitignore

```diff
@@ -337,4 +337,14 @@ ASALocalRun/
 .localhistory/
 
 # BeatPulse healthcheck temp database
-healthchecksdb
\ No newline at end of file
+healthchecksdb
+/.vscode/launch.json
+/.vscode/tasks.json
+/OtripleS.Web.Api/Migrations/20200719090247_AddCourses.cs
+/OtripleS.Web.Api/Migrations/20200719090247_AddCourses.Designer.cs
```

Comment:
Why do we want to ignore our migration files?

Reply:
@hassanhabib I had these files in my project and showing some changes that why i have ignored, and I wanted to submit my won changes only.

Reply:
@tajwal you need to pull a fresh copy from master from this repo then apply your changes.